tip-distributor/src/reclaim_rent_workflow.rs

```diff
@@ -0,0 +1,134 @@
+use {
+    crate::send_transactions_with_retry,
+    anchor_lang::AccountDeserialize,
+    log::info,
+    solana_client::nonblocking::rpc_client::RpcClient,
+    solana_program::pubkey::Pubkey,
+    solana_sdk::{
+        signature::{Keypair, Signer},
+        transaction::Transaction,
+    },
+    std::{error::Error, time::Duration},
+    tip_distribution::{
+        sdk::{
+            derive_config_account_address,
+            instruction::{
+                close_claim_status_ix, close_tip_distribution_account_ix, CloseClaimStatusAccounts,
+                CloseClaimStatusArgs, CloseTipDistributionAccountArgs,
+                CloseTipDistributionAccounts,
+            },
+        },
+        state::{ClaimStatus, Config, TipDistributionAccount},
+    },
+};
+
+pub async fn reclaim_rent(
```

Comment:
can you add some info/debug logs here? getPA can take ~20-30s so having an idea of whats going on helps :)

Reply:
done